inference-engine/tests/functional/inference_engine/serialization/single_layer/deformable_convolution.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "shared_test_classes/single_layer/deformable_convolution.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+TEST_P(DeformableConvolutionLayerTest, Serialize) {
+    Serialize();
+}
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32, InferenceEngine::Precision::FP16};
```

Comment:
Since convolution spec was modified to support any numeric type, I think we should test integer precisions too

Reply:
Done [here](https://github.com/openvinotoolkit/openvino/pull/4312/commits/2fefe8926da65171f64a2dc679ab548415736213)